cdci_data_analysis/analysis/parameters.py

```diff
@@ -683,14 +661,14 @@ def __init__(self,value=None,E_units='keV',name=None):
         #wtform_dict = {'keV': FloatField}
 
         super(SpectralBoundary, self).__init__(value=value,
-                                   units=E_units,
-                                   check_value=self.check_energy_value,
-                                   name=name,
-                                   allowed_units=_allowed_units)
+                                               units=E_units,
+                                               check_value=self.check_energy_value,
+                                               name=name,allowed_units=_allowed_units)
```

Comment:
in two lines?

Reply:
I don't know why they're like this here on guthub...

Reply:
hm. They are not like this elsewhere? I did not comment on it on github: I review directly in vscode.
Of course, both are MS products, and MS has been recently in code synthesis, who knows!

Reply:
but I use pyCharm which is not evil-MS...

Reply:
I used it before but it's not eco-friendly, it burns  lot's of cpu power! For some reason.